### PR TITLE
DOC-10991-Review-Updates-SG

### PR DESCRIPTION
**Pull request by dinaabennett**
Review updates for SG

### CODE REVIEW

**Review by franrob-projects — APPROVED** [review]
👍 💯

File: modules/ROOT/pages/server-compatibility-collections.adoc [hunk 1]
```diff
@@ -28,10 +28,9 @@ Couchbase Server provides the backing data store for Sync Gateway.
 
 TIP: See: {compatibility--xref} for version compatibility information.
 
-Here we provide details on how Couchbase Server's Scopes and Collections feature relates to the Couchbase mobile ecosystem.
 
-Sync Gateway offers support for Couchbase Server's default scopes and collections ({server-collections-default--xref}).
-It does not currently support named scopes or collections ({server-collections-named--xref}).
+Sync Gateway supports scopes and collections ({server-collections-default--xref}, {server-collections-named--xref} ).
```
Comment (osfameron) [inline modules/ROOT/pages/server-compatibility-collections.adoc:32]: Haven't you just deleted `server-collections-named--xref` above?

**Review by osfameron — CHANGES_REQUESTED** [review]
Bit confused about `server-collections-named--xref` comment inline.

File: modules/ROOT/pages/server-compatibility-collections.adoc (hunk 1 above)
Comment (dinaabennett) [reply modules/ROOT/pages/server-compatibility-collections.adoc:32]: No, I left the links to Server as-is for now.

File: modules/ROOT/pages/_partials/_define_page_index.adoc [hunk 2]
```diff
@@ -47,7 +47,7 @@ endif::xref--pfx-sgw[]
 :server-collections--page: learn:data/scopes-and-collections.adoc
 :server-collections--xref: {svr--xref-7x0}{server-collections--page}[Couchbase Collections]
 :server-collections-default--xref: {svr--xref-7x0}{server-collections--page}#default-scope-and-collection[Default Collections]
-:server-collections-named--xref: {svr--xref-7x0}{server-collections--page}#naming-for-scopes-and-collections[Named Collections]
```
Comment (osfameron) [inline modules/ROOT/pages/_partials/_define_page_index.adoc]: @dinaabennett ^ this is where it looks like that same ref has been deleted?

**Review by osfameron — APPROVED** [review]
Woot! Thanks Dina 👍

**Pull request merged**